### ISSUE TITLE
[Store][Bug] Users cannot download app from the store

### ISSUE THREAD

**Issue by MadMonkey87** [athombv/apps-sdk-issues#162]
Hi
Multiple  users have reported that they cannot install my app from the store. It seems homey is downloading (led indicator) but the app never appear. As this seems to be a store issue itself I do not have any log or any clue how this can be caused by the app.

Please also see the community discussion at https://community.athom.com/t/app-advanced-rest-client-arc/39662

**Comment by Dijker** [comment]
No problem installing it here on my Homey 5.0.0-rc.50 both the stable as the /test release.

**Comment by robertklep** [comment]
It fails to run using the CLI on a v4.2.0 Homey:
```
$ homey app run
✓ Pre-processing app...
✓ Running plugins...
✓ Running plugin `compose`...
[AppPluginCompose] Added FlowCard `request_completed` for type `triggers`
[AppPluginCompose] Added FlowCard `request_failed` for type `triggers`
[AppPluginCompose] Added FlowCard `header_collection_add` for type `actions`
[AppPluginCompose] Added FlowCard `header_collection_remove` for type `actions`
[AppPluginCompose] Added FlowCard `perform_request` for type `actions`
✓ Plugin `compose` finished
✓ Validating app...
✓ Homey App validated successfully against level `debug`
✓ Packing Homey App...
✓ Pruning dev dependencies...
 — App size: 1.14 MB
✓ Installing Homey App on `Homey` (https://192-168-23-8.homey.homeylocal.com)...
✖ Missing [[body]] in flow.actions[2].titleFormatted.en
```

If I add all the arguments to `titleFormatted`, it runs:
```
"titleFormatted": {
  "en": "[[verb]] [[url]] with body [[bodytype]] [[body]] [[headercollection]] [[certificate]]"
}
```

**Comment by robertklep** [comment]
@jeroenwienk do I understand correctly that the fix is to fail validation if not all arguments are used in `titleFormatted`? Or is that just done temporarily until it's fixed in the SDK?

**Comment by jeroenwienk** [comment]
> @jeroenwienk do I understand correctly that the fix is to fail validation if not all arguments are used in `titleFormatted`? Or is that just done temporarily until it's fixed in the SDK?

Yes all arguments should be used if titleFormatted is defined. This should be fixed in 2.9.4 which will be live any moment.
